### PR TITLE
Rollback sc-agent, sc-module and sc-keynodes

### PR DESCRIPTION
**Pull request by FalseR20**
And mark them as deprecated

### CODE REVIEW

File: src/sc_client/sc_agent.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import warnings
+from abc import ABC, abstractmethod
+
+from sc_client import client
+from sc_client.models import ScAddr, ScEvent
+from sc_client.sc_keynodes import ScKeynodes
+
+
+class ScAgent(ABC):
```
Comment (NikitaZotov) [inline src/sc_client/sc_agent.py:15]: There are tests for this logic?

Comment (FalseR20) [reply src/sc_client/sc_agent.py:15]: There were no tests even before removing (joke?).

Comment (FalseR20) [reply src/sc_client/sc_agent.py:15]: I don't want to copy tests from kpm

Comment (AlexandrZagorskiy) [reply src/sc_client/sc_agent.py:15]: Agreed, we will remove this untested logic, so there is no sense to make it now, just waste of time as I see

**Review by AlexandrZagorskiy — APPROVED** [review]

File: src/sc_client/sc_agent.py (hunk 1 above)
Comment (NikitaZotov) [reply src/sc_client/sc_agent.py:15]: Okay

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**